features/previousreleases.feature

```diff
@@ -1,33 +1,51 @@
+@previousReleases
 Feature: Previous Releases
 
   Scenario: GET /previousreleases and checking for one result
     Given there is a Search API that gives a successful response and returns 1 results
     And get page data request to zebedee for "/economy/latest" returns a page of type "article" with status 200
+    And get breadcrumb request to zebedee for "/economy/latest" returns breadcrumbs 
     And the search controller is running
     When I navigate to "/economy/previousreleases"
     Then the page should have the following content
     """
     {
-      "#main h1": "Previous releases for labour market statistics",
-      ".search__count h2": "1 result",
-      ".ons-pagination__position": "Page 1 of 1"
+      "#main h1": "labour market statistics",
+      ".ons-pagination__position": "Page 1 of 1",
+      ".ons-breadcrumb": "Home  Economy  Gross Domestic Product (GDP)  labour market statistics  Previous releases"
     }
     """
 
   Scenario: GET /previousreleases and checking for many results
     Given there is a Search API that gives a successful response and returns 3 results
     And get page data request to zebedee for "/economy/latest" returns a page of type "bulletin" with status 200
+    And get breadcrumb request to zebedee for "/economy/latest" returns breadcrumbs
     And the search controller is running
     When I navigate to "/economy/previousreleases"
     Then the page should have the following content
       """
       {
-        "#main h1": "Previous releases for labour market statistics",
-        ".search__count h2": "3 results",
-        ".ons-pagination__position": "Page 1 of 1"
+        "#main h1": "labour market statistics",
+        ".ons-pagination__position": "Page 1 of 1",
+        ".ons-breadcrumb": "Home  Economy  Gross Domestic Product (GDP)  labour market statistics  Previous releases"
       }
       """
 
+  Scenario: GET /previousreleases and breadcrumb request errors
+  Given there is a Search API that gives a successful response and returns 3 results
+  And get page data request to zebedee for "/economy/latest" returns a page of type "bulletin" with status 200
+  And get breadcrumb request to zebedee for "/economy/latest" fails
+  And the search controller is running
+  When I navigate to "/economy/previousreleases"
+  Then the page should have the following content
+    """
+    {
+      "#main h1": "labour market statistics",
+      ".ons-pagination__position": "Page 1 of 1",
+      ".ons-breadcrumb": "labour market statistics  Previous releases"
```

Comment:
Fascinating stuff.

Reply:
This is because we map the last two breadcrumbs with data we already have after getting the breadcrumb response whether the response is successful or not. 

I figured a breadcrumb link back to the bulletin might be useful in the case where the rest of the breadcrumb isn't available. No strong opinion on this and happy to change if desired